### PR TITLE
adding my profile in eddie community 

### PR DESCRIPTION
**Pull request by iamsunil25**

### CODE REVIEW

File: public/data/sunil_joshi.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "url": "https://github.com/iamsunil25",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/iamsunil25",
+      "icon": "twitter"
+    }
+  ]
+
```
Comment (CodingSpecies) [inline public/data/sunil_joshi.json:17]: Please can you remove this line `17` @iamsunil25 ? Thank you

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/sunil_joshi.json (hunk 1 above)
Comment (iamsunil25) [reply public/data/sunil_joshi.json:17]: yes i am removing

Comment (iamsunil25) [reply public/data/sunil_joshi.json]: @CodingSpecies i removed line 17

**Review by FayasNoushad — APPROVED** [review]

**Review by rohan-kulkarni-25 — APPROVED** [review]

**Review by Kunal0007 — APPROVED** [review]

**Review by CodingSpecies — APPROVED** [review]

File: public/data/sunil_joshi.json (hunk 1 above)
Comment (CodingSpecies) [reply public/data/sunil_joshi.json]: Thank you @iamsunil25

**Comment by iamsunil25** [comment]
@eddiejaoude can you please merge my pr

File: public/data/sunil_joshi.json (hunk 1 above)
Comment (krishguptadev) [reply public/data/sunil_joshi.json]: Please mark this as resolved if resolved

**Review by krishguptadev — APPROVED** [review]
LGTM 🚀 RR: @schmelto

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**